packages/styles/src/modules/typography.css

```diff
@@ -0,0 +1,90 @@
+/**
+ * Core typographic styles
+ */
+
+/**
+ * Utility class for setting a very small body copy font style
+ * May be combined with weight classes defined below
+ */
+.syn-body-x-small {
+  font: var(--syn-body-x-small-regular);
+}
+
+/**
+ * Utility class for setting a small body copy font style
+ * May be combined with weight classes defined below
+ */
+.syn-body-small {
+  font: var(--syn-body-small-regular);
+}
+
+/**
+ * Utility class for setting a medium body copy font style
+ * This is the default style for elements for synergy applications
+ * May be combined with weight classes defined below
+ */
+.syn-body-medium {
+  font: var(--syn-body-medium-regular);
+}
+
+/**
+ * Utility class for setting a large body copy font style
+ * May be combined with weight classes defined below
+ */
+.syn-body-large {
+  font: var(--syn-body-large-regular);
+}
+
+/**
+ * Utility class for setting a large headline
+ * This is usually seen on smaller weight headline elements as <h5> or <h6>
+ */
+.syn-heading-large {
+  font: var(--syn-heading-large);
+}
+
+/**
+ * Utility class for setting a extra large headline
+ * This is usually seen on medium weight headline elements as <h4> or <h3>
+ */
+.syn-heading-x-large {
+  font: var(--syn-heading-x-large);
+}
+
+/**
+ * Utility class for setting a even larger headline
+ * This is usually seen on larger weight headline elements as <h2> or <h3>
+ */
+.syn-heading-2x-large {
+  font: var(--syn-heading-2x-large);
+}
+
+/**
+ * Utility class for setting the largest headline
+ * This is usually seen on primary headline elements, usually <h1>
+ */
+.syn-heading-3x-large {
+  font: var(--syn-heading-3x-large);
+}
+
+/**
+ * Defines a font weight of normal (e.g. default font-weight for body copy), according to synergy specs
+ */
+ .syn-weight-normal {
+  font-weight: var(--syn-font-weight-normal);
+}
+
+/**
+ * Defines a font weight of semi-bold, according to synergy specs
+ */
+.syn-weight-semibold {
+  font-weight: var(--syn-font-weight-semibold);
+}
+
+/**
+ * Defines a font weight of bold, according to synergy specs
+ */
+.syn-weight-bold {
+  font-weight: var(--syn-font-weight-bold);
+}
```

Comment:
Do you think we need `weight` here in the name? Wouldn't it be more straightforward for a utility class to abandon it? I know, that syn-normal looks a bit strange and should be more `syn-regular`. (Maybe that's a problem in general.)

No blocker – just a thought.

Reply:
I also like `syn-bold` and `syn-semibold`. It's straightforward.

Reply:
I don´t have any problems with this. Classnames will get rather long if we prefix everything. When using `bold` or `semibold` it is indeed shorter and descriptive enough. I am not so sure that I would detect the meaning of `<div class="syn-regular">Hello World</div>` on my first glimpse of a piece of code.

Reply:
Yes it it. However you may run into something like:

```html
<p>
  <strong>Most of this sentence is bold, <span class="syn-regular">but this part is not</span></strong>
</p>
```

In this case, you would have to use the "regular" class.